src/elements/paginator/paginator.scss

```diff
@@ -146,7 +147,32 @@
     }
   }
 
-  &:hover {
+  &:disabled {
+    --sbb-paginator-page-color: var(--sbb-color-smoke);
+    --sbb-paginator-page-border-width: var(--sbb-border-width-1x);
+    --sbb-paginator-page-border-style: dashed;
+    --sbb-paginator-page-border-color: var(--sbb-color-smoke);
+    --sbb-paginator-page-background-color: var(--sbb-color-milk);
+    --sbb-paginator-page-cursor: default;
+
+    &[data-selected] {
+      --sbb-paginator-page-border-width: var(--sbb-border-width-2x);
```

Comment:
hmm I think border-width should only belong to data-selected. would we need an !important? or what is the main problem here?

Reply:
Since the disabled rule come after the data-selected, it overrides it
i can add a !important at line 134 to remove this line

Reply:
I'ts only a little bad design for me because the key value (width-2x) is assigend at two different places. otherwise an additional variable could be used. but maybe important solves it best, or a restructure ?

Reply:
I possibly fixed it by negating the condition for the -1x border